code/game/objects/structures/flora.dm

```diff
@@ -78,6 +78,15 @@
 	icon = 'icons/obj/plants.dmi'
 	icon_state = "plant-26"
 
+//Added random icon selection for potted plants.
+//It was silly they always used the same sprite when we have 26 sprites of them in the icon file
+/obj/structure/flora/pottedplant/New()
+	..()
+	var/number = rand(1,26)

```

Comment:
This will completely break if the dmi is changed, no? If so can you figure out a better way of doing it